frontend/projects/ui/src/app/pages/apps-routes/app-show/app-show.page.html

```diff
@@ -52,21 +52,17 @@
           <ion-row class="ion-align-items-center">
             <ion-col class="ion-text-center">
               <h2>
-                <ion-text color="warning">
-                  You are using an unencrypted http connection
-                </ion-text>
+                <ion-text color="warning">Http detected</ion-text>
```

Comment:
Haha I think this language is better....but 😄 
```suggestion
                <ion-text color="warning">Unencrypted connection</ion-text>
```